pom.xml

```diff
@@ -73,6 +73,9 @@
     <modules>
         <module>jsinterop-ts-defs-processor</module>
         <module>jsinterop-ts-defs-annotations</module>
+        <module>jsinterop-ts-defs-shared</module>
```

Comment:
Aside for "core" or "shared", i'll take just about any other suffix... I'd currently lean towards "impl" as a module name and sub-package, but maybe "writer", "transformer", etc?

Reply:
I kept this one unresolved even thought I changed to impl just in case we need to discuss this further